### DISCUSSION TITLE
Paperless with traefik always redirects to the admin interface

### DISCUSSION THREAD

**Discussion by Nutellaeis** [jonaswinkler/paperless-ng#878] [Support] [answered]
I am trying to run paperless-ng behind a traefik reverse proxy and it also kinda works but everytime I try to access paperless.somewhere.duckdns.org I end up at paperless.somewehere.duckdns.org/admin/ instead of paperless.somewehere.duckdns.org/dashboard/. Can someone point me in the right direction of what I did wrong?

Paperless part of docker compose:

https://pastebin.com/XYTdKx9r

**Comment by jonaswinkler** [comment, marked as answer]
Did you migrate from paperless?

Clear all your browsing data of paperless, old paperless installed a  permanent redirect in your browser.

**Reply by Nutellaeis** [reply]
That helped. I did not migrate from paperless but I did try it out in the past. Thank you for the quick help.
